src/hooks/audioSdk.ts

```diff
@@ -22,7 +22,8 @@ export const useMicrophoneTrack = createMicrophoneAudioTrack();
 export function useAudioSdk(
   channelName: string,
   ready: boolean,
-  track: IMicrophoneAudioTrack | null
+  track: IMicrophoneAudioTrack | null,
+  username: string | null
```

Comment:
Your call of useAudioSdk guarantees that username will not be null right? So you can remove the null from the typing